### ISSUE TITLE
Example fails - session evicted, reason=release_too_low

### ISSUE THREAD

**Issue by dyc3** [zetanumbers/tigerbeetle-rs#15]
Hey, I just tried to run the example in this repo. This was the output I got:

Notably:
```
error(client): 314031230543790906892500424223350319419: session evicted: reason=release_too_low (cluster_release=0.15.3)
thread 4121244 panic: session evicted
```

Full output
```
TigerBeetle C Sample
Connecting...
debug(tb_client_context): 314031230543790906892500424223350319419: init: initializing
debug(tb_client_context): 314031230543790906892500424223350319419: init: allocating tb_packets
debug(tb_client_context): 314031230543790906892500424223350319419: init: parsing vsr addresses: 6001
debug(tb_client_context): 314031230543790906892500424223350319419: init: initializing IO
debug(tb_client_context): 314031230543790906892500424223350319419: init: initializing MessagePool
debug(tb_client_context): 314031230543790906892500424223350319419: init: initializing client (cluster_id=00000000000000000000000000000000, addresses={ 127.0.0.1:6001 })
debug(vsr): 314031230543790906892500424223350319419: ping_timeout started
debug(tb_client_context): 314031230543790906892500424223350319419: init: initializing thread
debug(tb_client_thread): 314031230543790906892500424223350319419: init: initializing signal
debug(tb_client_thread): 314031230543790906892500424223350319419: init: spawning thread
debug(message_bus): connecting to replica 0 in 92ms...
debug(client): 314031230543790906892500424223350319419: register: registering a session with the cluster
debug(client): 314031230543790906892500424223350319419: send_request_for_the_first_time: request=0 checksum=327942018894883226467037249978675564056
debug(vsr): 314031230543790906892500424223350319419: request_timeout started
debug(client): 314031230543790906892500424223350319419: sending request to replica 0: vsr.message_header.Header{ .checksum = 327942018894883226467037249978675564056, .checksum_padding = 0, .checksum_body = 98287347720187652707502696638535748739, .checksum_body_padding = 0, .nonce_reserved = 0, .cluster = 0, .size = 256, .epoch = 0, .view = 0, .release = 0.0.1, .protocol = 0, .command = vsr.Command.request, .replica = 0, .reserved_frame = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, .reserved_command = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 59, 233, 123, 150, 57, 188, 84, 160, 52, 4, 10, 182, 12, 55, 64, 236, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 } }
debug(client): 314031230543790906892500424223350319419: request: user_data=1742584075557051830725066164993024 request=1 size=512 create_accounts
debug(message_bus): connecting to replica 0...
info(message_bus): connected to replica 0
debug(client): 314031230543790906892500424223350319419: on_message: vsr.message_header.Header{ .checksum = 129958897008263369093857830360855181532, .checksum_padding = 0, .checksum_body = 98287347720187652707502696638535748739, .checksum_body_padding = 0, .nonce_reserved = 0, .cluster = 0, .size = 256, .epoch = 0, .view = 4, .release = 0.15.3, .protocol = 0, .command = vsr.Command.eviction, .replica = 0, .reserved_frame = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, .reserved_command = { 59, 233, 123, 150, 57, 188, 84, 160, 52, 4, 10, 182, 12, 55, 64, 236, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2 } }
error(client): 314031230543790906892500424223350319419: session evicted: reason=release_too_low (cluster_release=0.15.3)
thread 4121244 panic: session evicted
/home/carson/Documents/code/other/tigerbeetle-rs/target/debug/build/tigerbeetle-unofficial-sys-3b89e3ba8357589c/out/tigerbeetle/src/vsr/client.zig:556:13: 0x55ea8098c445 in on_eviction (tb_client)
            @panic("session evicted");
            ^
/home/carson/Documents/code/other/tigerbeetle-rs/target/debug/build/tigerbeetle-unofficial-sys-3b89e3ba8357589c/out/tigerbeetle/src/vsr/client.zig:276:50: 0x55ea8097d689 in on_message (tb_client)
                .eviction => |m| self.on_eviction(m),
                                                 ^
/home/carson/Documents/code/other/tigerbeetle-rs/target/debug/build/tigerbeetle-unofficial-sys-3b89e3ba8357589c/out/tigerbeetle/src/message_bus.zig:813:40: 0x55ea80a6c4b2 in on_message (tb_client)
                bus.on_message_callback(bus, message);
                                       ^
/home/carson/Documents/code/other/tigerbeetle-rs/target/debug/build/tigerbeetle-unofficial-sys-3b89e3ba8357589c/out/tigerbeetle/src/message_bus.zig:701:42: 0x55ea80a6b8cc in parse_messages (tb_client)
                    connection.on_message(bus, message);
                                         ^
/home/carson/Documents/code/other/tigerbeetle-rs/target/debug/build/tigerbeetle-unofficial-sys-3b89e3ba8357589c/out/tigerbeetle/src/message_bus.zig:943:42: 0x55ea80a6b7e0 in on_recv (tb_client)
                connection.parse_messages(bus);
                                         ^
/home/carson/Documents/code/other/tigerbeetle-rs/target/debug/build/tigerbeetle-unofficial-sys-3b89e3ba8357589c/out/tigerbeetle/src/io/linux.zig:808:29: 0x55ea80a6b254 in wrapper (tb_client)
                    callback(
                            ^
/home/carson/Documents/code/other/tigerbeetle-rs/target/debug/build/tigerbeetle-unofficial-sys-3b89e3ba8357589c/out/tigerbeetle/src/io/linux.zig:545:28: 0x55ea8098ebc9 in call_callback (tb_client)
        completion.callback(completion.context, completion, result);
                           ^
/home/carson/Documents/code/other/tigerbeetle-rs/target/debug/build/tigerbeetle-unofficial-sys-3b89e3ba8357589c/out/tigerbeetle/src/io/linux.zig:449:34: 0x55ea8098039f in complete (tb_client)
                    call_callback(completion, &result, callback_tracer_slot);
                                 ^
/home/carson/Documents/code/other/tigerbeetle-rs/target/debug/build/tigerbeetle-unofficial-sys-3b89e3ba8357589c/out/tigerbeetle/src/io/linux.zig:150:70: 0x55ea8097f5db in flush (tb_client)
        while (self.completed.pop()) |completion| completion.complete(&self.callback_tracer_slot);
                                                                     ^
/home/carson/Documents/code/other/tigerbeetle-rs/target/debug/build/tigerbeetle-unofficial-sys-3b89e3ba8357589c/out/tigerbeetle/src/io/linux.zig:110:27: 0x55ea809f9c15 in run_for_ns (tb_client)
            try self.flush(1, &timeouts, &etime);
                          ^
/home/carson/Documents/code/other/tigerbeetle-rs/target/debug/build/tigerbeetle-unofficial-sys-3b89e3ba8357589c/out/tigerbeetle/src/clients/c/tb_client/context.zig:236:35: 0x55ea809e32bf in run (tb_client)
                self.io.run_for_ns(constants.tick_ms * std.time.ns_per_ms) catch |err| {
                                  ^
/home/carson/Documents/code/other/tigerbeetle-rs/target/debug/build/tigerbeetle-unofficial-sys-3b89e3ba8357589c/out/tigerbeetle/zig/lib/std/Thread.zig:412:13: 0x55ea809ce6ea in callFn__anon_47629 (tb_client)
            @call(.auto, f, args);
            ^
/home/carson/Documents/code/other/tigerbeetle-rs/target/debug/build/tigerbeetle-unofficial-sys-3b89e3ba8357589c/out/tigerbeetle/zig/lib/std/Thread.zig:685:30: 0x55ea8098f3a2 in entryFn (tb_client)
                return callFn(f, args_ptr.*);
                             ^
./nptl/pthread_create.c:442:8: 0x7f78a27bfac2 in start_thread (pthread_create.c)
../sysdeps/unix/sysv/linux/x86_64/clone3.S:81:0: 0x7f78a285184f in ??? (../sysdeps/unix/sysv/linux/x86_64/clone3.S)
???:?:?: 0x0 in ??? (???)
[1]    4121241 IOT instruction (core dumped)  TB_ADDRESS="6001" cargo run --example c_port_high_level
```

This is my docker-compose, which works for other clients:

```
version: '3.8'

services:
  tigerbeetle:
    image: ghcr.io/tigerbeetle/tigerbeetle:0.15.3
    command: ["start", "--addresses=0.0.0.0:6001", "/data/0_0.tigerbeetle"]
    volumes:
      - ./data/tigerbeetle:/data
    ports:
      - "6001:6001"
    restart: unless-stopped
```

**Comment by dyc3** [comment]
These are the logs from tigerbeetle:
```
tigerbeetle_1  | info(io): opening "0_0.tigerbeetle"...
tigerbeetle_1  | info(main): 0: Allocated 4274MiB in 16 regions during replica init
tigerbeetle_1  | info(main): 0: Grid cache: 1024MiB, LSM-tree manifests: 128MiB
tigerbeetle_1  | info(main): 0: cluster=0: listening on 0.0.0.0:6001
tigerbeetle_1  | warning(main): 0: started with constants.verify - expect reduced performance. Recompile with -Dconfig=production if unexpected.
tigerbeetle_1  | info(message_bus): connection from client 314031230543790906892500424223350319419
tigerbeetle_1  | warning(replica): 0: on_request: ignoring invalid version (client=314031230543790906892500424223350319419 version=0.0.1<0.15.3)
tigerbeetle_1  | error(replica): 0: sending eviction message to client=314031230543790906892500424223350319419 reason=release_too_low
tigerbeetle_1  | info(message_bus): peer performed an orderly shutdown: message_bus.MessageBusType(.replica).Connection.Peer{ .client = 314031230543790906892500424223350319419 }
```

**Comment by zetanumbers** [comment]
Fixed by a096549456e868cc6d46901c13c0d892e443942e

**Comment by zetanumbers** [comment]
Fix published in 0.4.1+0.15.3
